doc/releases.md

```diff
@@ -95,55 +118,46 @@ See https://github.com/nodejs/node/labels/confirmed-bug for complete and current
 
 ### Commits
 
-* Include the full list of commits since the last release here
+* Include the full list of commits since the last release here. Do not include "Working on X.Y.Z+1" commits.
 ```
 
-### 4. Update _src/node_version.h_
+The release type should be either Stable, LTS, or Maintenance, depending on the type of release being produced.
```

Comment:
Should likely note that only the LTS WG can cut LTS releases and that there is a slightly modified process that uses the various `-staging` branches